### ISSUE TITLE
[Suggestion] Replace creeperSpawningInJungleTemples with a different mob

### ISSUE THREAD

**Issue by God-ly** [gnembon/carpet-extra#183]
It's already possible to use trapdoors and carpets to make a space where only creepers can spawn, which makes this feature replaceable in vanilla. There are four alternatives to creepers I can think of:
- Skeleton: There is no way to exclusively spawn skeletons in vanilla (strays in igloos is the closest match in this mod currently), so skeletons could be a good way to fit the theme of the jungle temple, as one of its traps shoot out arrows.
- Zombie: There is also no way to exclusively spawn skeletons in vanilla (husks in desert temples is the closest match in this mod currently), though zombies don't really fit the desert temple too well.
- Spider: Jungle and its related biomes already give a higher spawn weight to spiders, so it would fit the theme of the jungle trees to have spiders in the jungle temple. However, spiders usually don't have enough space to spawn and it doesn't really fit the structure. Also, it is already possible to make a spider-exclusive farm
- Spider Jockey: There isn't any way to exclusively spawn spider jockeys in vanilla nor this mod. If this were added, it would fit the theme of both the jungle and the temples. It would also probably fill the surrounding area with spider jockeys, as if they were guarding the entrance.
I definitely like the last option the best if spider jockeys can spawn at decent rates inside an unmodified jungle temple, but they might not be able to have enough room. Still, I think any of the four options are better than creepers.

**Comment by DragonEggBedrockBreaking** [comment]
>     * Skeleton: There is no way to exclusively spawn skeletons in vanilla (strays in igloos is the closest match in this mod currently), so skeletons could be a good way to fit the theme of the jungle temple, as one of its traps shoot out arrows.

Soul sand valley + two high space so no ghasts
 
>     * Zombie: There is also no way to exclusively spawn skeletons in vanilla (husks in desert temples is the closest match in this mod currently), though zombies don't really fit the desert temple too well.

Pointless, because their only drops are either rare and farmable (e.g. carrot, potato, iron) or useless and available in husks (rotten flesh).

>     * Spider: Jungle and its related biomes already give a higher spawn weight to spiders, so it would fit the theme of the jungle trees to have spiders in the jungle temple. However, spiders usually don't have enough space to spawn and it doesn't really fit the structure. Also, it is already possible to make a spider-exclusive farm

As you said, spider farms already exist.

>     * Spider Jockey: There isn't any way to exclusively spawn spider jockeys in vanilla nor this mod. If this were added, it would fit the theme of both the jungle and the temples. It would also probably fill the surrounding area with spider jockeys, as if they were guarding the entrance.

This is imo the best option, and it would nicely complement the spider jokeys drop gapples rule.

**Comment by God-ly** [comment]
Yeah, the last option has the most uniqueness to it, only small problem is its spawn rates with an unmodified temple. Stray farms are much faster than soul sand valley farms due to the spawn mechanics in the nether and strays also give the exact same loot but a little bit better. Every single item that a zombie drops can be obtained in a more efficient way. I just put that many options there in case I overlooked something.

**Comment by DragonEggBedrockBreaking** [comment]
> Yeah, the last option has the most uniqueness to it, only small problem is its spawn rates with an unmodified temple.
But why would you use an unmodified temple for any farm, whether it be spider, zombie, skeleton, creeper, etc.

**Comment by God-ly** [comment]
Part of carpet extra is adding enhancement to survival gameplay like when you see the temple for the first time. It would be more of a challenge convert the temple if more spider jockeys were able to spawn there in the very beginning.

**Comment by DragonEggBedrockBreaking** [comment]
> Part of carpet extra is adding enhancement to survival gameplay like when you see the temple for the first time. It would be more of a challenge convert the temple if more spider jockeys were able to spawn there in the very beginning. Anyways, hope gnembon sees this.

Well a solution to this would also be for spider jockeys to generate in the temple on first generation, and also to spawn later on when the conditions are met. Also, changing spawning conditions could be a complementary solution, e.g. for (carpet's) husks spawning in desert temples, husks no longer need an open sky to spawn, if they spawn in the temples.
